### PR TITLE
Create initial meetings plugin scaffolding

### PR DESCRIPTION
**Pull request by mattnorris**
Rebased and addressed comments on Bernie's PR, #939

### CODE REVIEW

File: webpack/webpack.config.base.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          loader: 'babel-loader'
+        }
+      }
+    ]
+  },
+  plugins: [
+    new webpack.EnvironmentPlugin({
+      CISCOSPARK_LOG_LEVEL: 'log',
+      DEBUG: '',
+      NODE_ENV: process.env.NODE_ENV || 'development',
```
Comment (mattnorris) [inline webpack/webpack.config.base.js]: @adamweeks

File: package.json [hunk 2]
```diff
@@ -162,6 +163,7 @@
     "grunt-env": "~0.4.2",
     "grunt-gh-pages": "~2.0.0",
     "highlight.js": "^9.5.0",
+    "html-webpack-plugin": "^3.2.0",
```
Comment (mattnorris) [inline package.json]: @adamweeks

File: packages/node_modules/@webex/plugin-meetings/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+## Usage
+
+This is a plugin for the Cisco Webex JS SDK . Please see our [developer portal](https://developer.webex.com/sdks-and-widgets.html) and the [API docs](https://webex.github.io/spark-js-sdk/api/) for full details.
+
+## Development
+
+To use `webpack-dev-server` to load this package, run `npm run serve:package -- --env.package @webex/plugin-meetings` from the root of this repo.
+
+Files placed in the `samples` folder will be served statically.
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/README.md]: This should just be `sample`

File: packages/node_modules/@webex/plugin-meetings/package.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@webex/plugin-meetings",
+  "version": "1.32.28",
+  "description": "",
+  "license": "MIT",
+  "contributors": [
+    "Bernie Zang <nzang@cisco.com>"
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/package.json]: Want to add the team?

**Pull request merged**